### PR TITLE
Include MOTD feature in CI tests #7340

### LINKED ISSUE #7340: Include MOTD feature in CI tests
Current:
* Message of the Day feature is omitted in CI tests i.e. the relevant property is left blank in the `build.properties` file

Suggested:
* Include MOTD feature in tests so that problems like this #7335 can be detected by CI

### PR DESCRIPTION
**Pull request by whipermr5**
Fixes #7340

**Outline of Solution**

- Add default MOTD URL to `build.properties`

### CODE REVIEW

File: src/test/testng-ci.xml [hunk 1]
```diff
@@ -50,6 +50,7 @@
     <test name="parallel-ui-tests" parallel="classes">
         <!-- These cover the important UI functionality. Can be run in parallel. -->
         <classes>
+            <class name="teammates.test.cases.browsertests.GodModeTest" />
```
Comment (damithc) [inline src/test/testng-ci.xml]: Is this related to this PR?

File: build.gradle [hunk 2]
```diff
@@ -167,22 +167,27 @@ eclipseClasspath {
     }
 }
```
Comment (damithc) [inline build.gradle]: If somebody accidentally deleted the MOTD placeholder (like what happened recently), would that be detected after this PR is applied?

File: src/test/testng-ci.xml (hunk 1 above)
Comment (whipermr5) [inline src/test/testng-ci.xml]: I was using it to verify that the MOTD URL is defined, though on second thoughts, that's not really necessary. I also just realised that this PR allows GodMode enabled on Travis without failing the build 😅 Fixing.

File: build.gradle (hunk 2 above)
Comment (whipermr5) [inline build.gradle]: Yup [looks like it works](https://travis-ci.org/TEAMMATES/teammates/builds/235932910) :)
